services/cinema_hall.py

```diff
@@ -0,0 +1,17 @@
+import init_django_orm  # noqa: F401
+
+from db.models import CinemaHall
+
+
+def get_cinema_halls() -> CinemaHall:
```

Comment:
Invalid return type. It returns queryset of CinemaHall
```suggestion
def get_cinema_halls() -> QuerySet[CinemaHall]:
```